### PR TITLE
Release notes for 2.18.1

### PR DESCRIPTION
**Pull request by bunchesofdonald**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+
+#### Prefect UI Library
+- Update date-fns and date-fns-tz — https://github.com/PrefectHQ/prefect-ui-library/pull/2319
+- Enhancement: Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
+- Enhancement: flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
+- Fix "Run a deployment" action not populating the default parameters from the deployment — https://github.com/PrefectHQ/prefect-ui-library/pull/2322
+- Fix schema form properties with no default value from defaulting to `null` (`None`) — https://github.com/PrefectHQ/prefect-ui-library/pull/2323
+- Use correct icon colors for non-destructive actions.  — https://github.com/PrefectHQ/prefect-ui-library/pull/2328
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
+- Release 2.8.14 — https://github.com/PrefectHQ/prefect-ui-library/pull/2330
```
Comment (serinamarie) [inline RELEASE-NOTES.md:49]: oh this I think was autointegrated from the UI library. @collincchoy do you know if all these PRs are needed/if there are any more to include?

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+- Allow `prefect cloud login` to override current workspace — https://github.com/PrefectHQ/prefect/pull/12867
+- Remove extra quotes in `prefect deployment run --watch` — https://github.com/PrefectHQ/prefect/pull/12894
+
+### Experimental
+
+#### Events and Automations
+- Support filtering by automation name:
+    - https://github.com/PrefectHQ/prefect/pull/12850
+    - https://github.com/PrefectHQ/prefect/pull/12884
+    - https://github.com/PrefectHQ/prefect/pull/12887
```
Comment (serinamarie) [inline RELEASE-NOTES.md:18]: nice ❤️

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+- Add support for using the "normal" Trigger classes for flow.serve and .deploy — https://github.com/PrefectHQ/prefect/pull/12789
```
Comment (serinamarie) [inline RELEASE-NOTES.md:19]:
```suggestion
- Add support for using the "normal" Trigger classes for `flow.serve` and `.deploy` — https://github.com/PrefectHQ/prefect/pull/12789
```

what is meant by "normal" here?

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,64 @@
...
+- Add an account-level event subscriber — https://github.com/PrefectHQ/prefect/pull/12808
+- Emit flow run state change events — https://github.com/PrefectHQ/prefect/pull/12825
+- Emit deployment status persistence and events — https://github.com/PrefectHQ/prefect/pull/12853
+- Enable Event Streaming from PrefectCloudEventSubscriber via CLI  — https://github.com/PrefectHQ/prefect/pull/12796
+- Update `prefect automation delete` — https://github.com/PrefectHQ/prefect/pull/12876
```
Comment (serinamarie) [inline RELEASE-NOTES.md:24]:
```suggestion
- Update the `prefect automation delete` CLI — https://github.com/PrefectHQ/prefect/pull/12876
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,64 @@
...
+
+#### Engine
+- New experimental engine for tasks and flows with improved readability and extensibility — https://github.com/PrefectHQ/prefect/pull/12856
```
Comment (serinamarie) [inline RELEASE-NOTES.md:27]:
```suggestion
- Add new experimental engine for tasks and flows with improved readability and extensibility — https://github.com/PrefectHQ/prefect/pull/12856
```

is this worth mentioning? we don't usually highlight experimental stuff, but jw

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,64 @@
...
+
+### Documentation
+- Improve installation instructions  — https://github.com/PrefectHQ/prefect/pull/12783
+- Quickstart and other documentation improvements  — https://github.com/PrefectHQ/prefect/pull/12798
```
Comment (serinamarie) [inline RELEASE-NOTES.md:31]:
```suggestion
- Improve quickstart  — https://github.com/PrefectHQ/prefect/pull/12798
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,64 @@
...
+- Migrate `prefect-azure` docs to Integrations section of the Prefect docs — https://github.com/PrefectHQ/prefect/pull/12794
+- Update storage guide credentials blocks — https://github.com/PrefectHQ/prefect/pull/12819
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Better wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
```
Comment (serinamarie) [inline RELEASE-NOTES.md:37]:
```suggestion
- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,64 @@
 # Prefect Release Notes
 
+## Release 2.18.1
+
+### Fixes
+- Fix improper context access for nested async task outside of flow — https://github.com/PrefectHQ/prefect/pull/12810
+- Fix using default interval schedule in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12833
+- Fix a bug in the schema validationUpdate where definitions are falsy — https://github.com/PrefectHQ/prefect/pull/12880
```
Comment (serinamarie) [inline RELEASE-NOTES.md:8]:
```suggestion
- Handle cases in `validationUpdate` schema where definitions are falsy — https://github.com/PrefectHQ/prefect/pull/12880
```

File: RELEASE-NOTES.md (hunk 1 above)
Comment (serinamarie) [inline RELEASE-NOTES.md:49]:
```suggestion
```

Comment (serinamarie) [reply RELEASE-NOTES.md:49]: @bunchesofdonald we should probably order by importance too, I see a feature in there

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,64 @@
...
+#### Prefect CGP
+- Remove API ref to nonexistent Google Cloud Run V2 page — https://github.com/PrefectHQ/prefect-gcp/pull/260
+- Fix VPC access for Cloud v2 worker — https://github.com/PrefectHQ/prefect-gcp/pull/266
+- Handle case where `vpc` isn't in job template — https://github.com/PrefectHQ/prefect-gcp/pull/267
+
+## New Contributors
+* @keizobabybear made their first contribution in https://github.com/PrefectHQ/prefect/pull/12852
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.18.0...2.18.1
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:60]:
```suggestion
```

File: RELEASE-NOTES.md (hunk 1 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md:49]: Good call, I can do that.

File: RELEASE-NOTES.md (hunk 3 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md:19]: Not sure, I can look.

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,63 @@
...
+- Improve installation instructions  — https://github.com/PrefectHQ/prefect/pull/12783
+- Improve quickstart  — https://github.com/PrefectHQ/prefect/pull/12798
+- Migrate `prefect-azure` docs to Integrations section of the Prefect docs — https://github.com/PrefectHQ/prefect/pull/12794
+- Update storage guide credentials blocks — https://github.com/PrefectHQ/prefect/pull/12819
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+### Integrations
```
Comment (serinamarie) [inline RELEASE-NOTES.md:39]: I think this heading should go down above Prefect GCP

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,63 @@
...
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
+- Enhancement: Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
+- Enhancement: flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
+- Fix "Run a deployment" action not populating the default parameters from the deployment — https://github.com/PrefectHQ/prefect-ui-library/pull/2322
+- Fix schema form properties with no default value from defaulting to `null` (`None`) — https://github.com/PrefectHQ/prefect-ui-library/pull/2323
+- Update date-fns and date-fns-tz — https://github.com/PrefectHQ/prefect-ui-library/pull/2319
+- Use correct icon colors for non-destructive actions.  — https://github.com/PrefectHQ/prefect-ui-library/pull/2328
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
```
Comment (serinamarie) [inline RELEASE-NOTES.md:49]: this is a dupe

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,63 @@
...
+### Integrations
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
+- Enhancement: Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
+- Enhancement: flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
+- Fix "Run a deployment" action not populating the default parameters from the deployment — https://github.com/PrefectHQ/prefect-ui-library/pull/2322
+- Fix schema form properties with no default value from defaulting to `null` (`None`) — https://github.com/PrefectHQ/prefect-ui-library/pull/2323
+- Update date-fns and date-fns-tz — https://github.com/PrefectHQ/prefect-ui-library/pull/2319
+- Use correct icon colors for non-destructive actions.  — https://github.com/PrefectHQ/prefect-ui-library/pull/2328
```
Comment (serinamarie) [inline RELEASE-NOTES.md:48]:
```suggestion
- Use correct icon colors for non-destructive actions in the UI  — https://github.com/PrefectHQ/prefect-ui-library/pull/2328
```

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,63 @@
...
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+### Integrations
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
+- Enhancement: Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
+- Enhancement: flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
```
Comment (serinamarie) [inline RELEASE-NOTES.md:44]:
```suggestion
- Add flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
```

maybe?

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,63 @@
...
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+### Integrations
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
+- Enhancement: Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
```
Comment (serinamarie) [inline RELEASE-NOTES.md:43]:
```suggestion
- Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
```

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,63 @@
...
+- Update storage guide credentials blocks — https://github.com/PrefectHQ/prefect/pull/12819
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+### Integrations
+
+#### Prefect UI Library
+- Feature: `FormattedDate` component — https://github.com/PrefectHQ/prefect-ui-library/pull/2329
```
Comment (serinamarie) [inline RELEASE-NOTES.md:42]: Is there a way to make this sound more user-facing?

Comment (bunchesofdonald) [reply RELEASE-NOTES.md:42]: I can come up with something.

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,62 @@
...
+- Improve quickstart  — https://github.com/PrefectHQ/prefect/pull/12798
+- Migrate `prefect-azure` docs to Integrations section of the Prefect docs — https://github.com/PrefectHQ/prefect/pull/12794
+- Update storage guide credentials blocks — https://github.com/PrefectHQ/prefect/pull/12819
+- Remove `server` import recommendations — https://github.com/PrefectHQ/prefect/pull/12823
+- Remove link to removed API page — https://github.com/PrefectHQ/prefect/pull/12824
+- Add Azure Container Instances worker guide — https://github.com/PrefectHQ/prefect/pull/12846
+- Improve wording on integrations index page — https://github.com/PrefectHQ/prefect/pull/12852
+
+#### Prefect UI Library
+- `FormattedDate` component to display accessible, long-form timestamps consistently.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:40]:
```suggestion
- Add `FormattedDate` component to display accessible, long-form timestamps consistently
```

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -1,5 +1,62 @@
...
+- `FormattedDate` component to display accessible, long-form timestamps consistently. 
+- Update modal buttons and add auto-close to the parameters and job variable modals — https://github.com/PrefectHQ/prefect-ui-library/pull/2320
+- Add flow run list information density — https://github.com/PrefectHQ/prefect-ui-library/pull/2321
+- Fix "Run a deployment" action not populating the default parameters from the deployment — https://github.com/PrefectHQ/prefect-ui-library/pull/2322
+- Fix schema form properties with no default value from defaulting to `null` (`None`) — https://github.com/PrefectHQ/prefect-ui-library/pull/2323
+- Update date-fns and date-fns-tz — https://github.com/PrefectHQ/prefect-ui-library/pull/2319
+- Use correct icon colors for non-destructive actions in the UI  — https://github.com/PrefectHQ/prefect-ui-library/pull/2328
+
+### Integrations
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:49]:
```suggestion
```

**Review by serinamarie — APPROVED** [review]
I just committed those last 2 suggestions since they were very nit. LGTM!

**Comment by bunchesofdonald** [comment]
Thanks @serinamarie !

**Pull request merged**